content/plugins/overview.md

```diff
@@ -56,8 +56,9 @@ _Note: This method is available when working in a [Local CLI profile]({{< relref
 
 Plugin usage inside of blueprints varies. However, these two general rules apply:
 
-- Your blueprint must import the `plugin.yaml` of the pluigns you would like to use. See [importing]({{< relref "blueprints/spec-imports.md" >}}) plugins for more information. See also [spec-plugins]({{< relref "blueprints/spec-plugins.md" >}}).
-- Either the `plugin.yaml` or your blueprint will map node lifecycle operations to appropriate plugin functions.
+- Your blueprint must add import statement for the plugins that in use. See [importing]({{< relref "blueprints/spec-imports.md" >}}) plugins for more information
```

Comment:
add import statement for the plugins that in use > add an import statement for the plugins that are in use
In general, we don't remove words that seem unecessary